src/components/navbar/Navbar.js

```diff
@@ -46,76 +81,230 @@ function Navbar() {
           : "navbar__wrapper navbar__wrapper--scrolled"
       }
     >
-      <div className="navbar">
-        <Link
-          to="/"
-          className={
-            isScrolled === false
-              ? "Navbar__logo"
-              : "Navbar__logo Navbar__logo--scrolled"
-          }
-        >
-          <div>RStore</div>
-        </Link>
-        <div className="navbar__linkWrapper">
-          <ul>
-            <Link
-              to="/"
-              className={
-                isScrolled === false
-                  ? "navbar__Link"
-                  : "navbar__Link navbar__Link--scrolled"
-              }
-            >
-              <li
-                onClick={() =>
-                  dispatch({
-                    type: "PRODUCTS_TO_SHOW",
-                    payload: "AllProducts",
-                  })
-                }
+      {isSideMenuOpen && width < 770 && (
+        <div className="Navbar__sideMenu">
+          <nav className="Navbar__sideMenuNav">
+            <div className="Navbar__searchInputContainer">
+              <input
+                type="text"
+                className="Navbar__searchInput"
+                placeholder="Search..."
+                onChange={handleSearchInputChange}
+              />
+              <Link
+                to={{ pathname: "/search", state: { filteredProducts } }}
+                className="Navbar__searchInputIcon"
+                onClick={() => setIsSideMenuOpen(false)}
               >
-                AllProducts
-              </li>
-            </Link>
-
-            <Link
-              to="/"
-              className={
-                isScrolled === false
-                  ? "navbar__Link"
-                  : "navbar__Link navbar__Link--scrolled"
-              }
-            >
-              <li
-                onClick={() =>
-                  dispatch({ type: "PRODUCTS_TO_SHOW", payload: "Wishlist" })
-                }
+                <BiSearch className="Navbar__searchInputIcon" color="#6e6e73" />
+              </Link>
+            </div>
+            <div className="Navbar__sideMenu--linkWrapper">
+              <div
+                className="Navbar__sideMenu--linkConatiner"
+                style={{ marginTop: "32px" }}
```

Comment:
avoid inline style

Reply:
This only has to be used at one place that's why using inline